content/chat.adoc

```diff
@@ -81,10 +81,10 @@ We currently offer the following cloak levels:
 
 To obtain a cloak:
 
-* ensure you have a *registered user* on Freenode
+* ensure you have a *registered user* on Libera Chat
 * ensure you are already recognized as a developer (have +vV voice privilege)
 * email jieryn@gmail.com with those details
-* monitor your memoserv notices for updates from Freenode staff requesting confirmation of cloak request
+* monitor your memoserv notices for updates from Libera Chat staff requesting confirmation of cloak request
```

Comment:
not accurate, not sure exactly how to claim one but the docs are here: https://libera.chat/chanreg